comps/llms/text-generation/tgi/llm.py

```diff
@@ -82,16 +82,15 @@ async def stream_generator():
                 async for text in text_generation:
                     stream_gen_time.append(time.time() - start)
                     chat_response += text
-                    chunk_repr = repr(text.encode("utf-8"))
```

Comment:
Encoding is required by the frontend to prevent special characters from being omitted. Have you tried the Svelte UI? We need to ensure that both the React UI and Svelte UI work seamlessly, whether in Chinese or English, with your code changes.

Reply:
Thanks for the comments! Check with Svelte UI, it can display both Chinese and English well, for it has the logic to decode hexadecimal for Chinese response. It seems that modifying React UI involves minor changes.